rossum_api/models/__init__.py

```diff
@@ -78,6 +78,10 @@ def deserialize_default(resource: Resource, payload: JsonDict) -> Any:
     for example datetime.
     """
     model_class = RESOURCE_TO_MODEL[resource]
+
+    if resource == Resource.Schema:
+        return Schema.from_dict(payload)
```

Comment:
This if is suspicious, why do you need this if? `dacite` can de-serialize stuff recursively afaik.

Reply:
It is necessary to use correct class for deserialization based on `category` value, .e.g Datapoint, Multivalue etc... I didn't make dacite to work OOTB.